lago/utils.py

```diff
@@ -421,6 +421,10 @@ def deepcopy(original_obj):
     """
     if isinstance(original_obj, list):
         return list(deepcopy(item) for item in original_obj)
+    elif isinstance(original_obj, collections.OrderedDict):
+        return collections.OrderedDict(
+            (key, deepcopy(val)) for key, val in original_obj.items()
+        )
```

Comment:
Not an issue with this patch - but stdlib's copy.deepcopy will not work here?(replacing the entire function)